data/types/settings/pump/sleep_schedule.go

```diff
@@ -0,0 +1,120 @@
+package pump
+
+import (
+	"sort"
+
+	"github.com/tidepool-org/platform/data"
+	dataTypesCommon "github.com/tidepool-org/platform/data/types/common"
+	"github.com/tidepool-org/platform/structure"
+	structureValidator "github.com/tidepool-org/platform/structure/validator"
+)
+
+const (
+	SleepSchedulesMidnightOffsetMaximum = 86400
```

Comment:
I will check with Gerrit

Reply:
Its is currently seconds. Lets leave it as such, otherwise we would have to migrate the existing sleepschedule data adding more time to all of this process